src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -39,6 +39,19 @@ $weekday--margin-right: 10px;
     padding-left: $NavBarSchedule--padding;
   }
 
+  &__downloads {
+    display: flex;
+    cursor: pointer;
+    padding-left: $NavBarSchedule--padding;
+    opacity: 0.8;
+    text-shadow: 0 2px 4px rgba($black, 0.3);
+    box-shadow: 0 3px 6px rgba($black, 0.2);
+
+    &:hover {
+      opacity: 1;
+    }
+  }
```

Comment:
[note] I haven't deeply reviewed these styles visually, they look frome a code point of view though.